blockchain/fullblocktests/generate.go

```diff
@@ -525,17 +525,38 @@ func Generate(includeLargeReorg bool) (tests [][]TestInstance, err error) {
 	g.AssertTipHeight(1)
 	accepted()
 
-	// TODO:
-	// - Try to spend premine output before maturity in the regular tree
-	// - Try to spend premine output before maturity in the stake tree by
-	//   creating a ticket purchase
+	// Create block that tries to spend premine output before
+	// maturity in the regular tree
+	//
+	// .. bp -> bpi0
```

Comment:
The block is expected to be rejected, so it should be shown on the next line to be consistent.  Also, there aren't multiple blocks prior to it in order to warrant continuation dots (which incidentally would also be 3 dots for consistency if there were to be used).

```Go
// genesis -> bp
//              \-> bpi0
```